tutorials/beginner/execution_engine.ipynb

```diff
@@ -11,16 +11,58 @@
     "[![Homepage](https://img.shields.io/badge/fugue-source--code-red?logo=github)](https://github.com/fugue-project/fugue)\n",
```

Comment:
<p>You got some typo in the first sentence. </p><p> </p><p>Maybe <code>aims to make big data compute projects effortless</code>.</p><p> </p><p>Do we need the re-iteration though? I think the previous was good.</p><p> </p><blockquote>We have already seen that the <code>transform()</code> function takes in an <code>engine</code>. </blockquote><p> </p><p>I think this is good because it connects the current section to previously discussed stuff while the current first sentence in this PR does not. </p><p> </p><p>I think there is no need to make "Engine" in the code block. You can just use normal text "Default Engine". It just looks a big ugly.</p><p> </p><p>Suggestions for the bullets:</p><ul><li>Inferring the Engine (Default)</li><li>Passing the Engine as String</li><li>Passing the Engine as Client or Session</li><li>Using Hosted Clusters as the Engine</li><li>Engine Configurations</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fugue-project/tutorials/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>